packages/app/components/OrdersProgressBar.tsx

```diff
@@ -0,0 +1,20 @@
+import { OrderProps, totalOrdersDone, totalOrders } from "@/app/models/order";
+import React, { useRef, useEffect } from "react";
+
+export const OrdersProgressBar = ({ order }: OrderProps) => {
+  const progressBarRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (progressBarRef.current) {
+      const totalWidth = progressBarRef.current.offsetWidth;
+      const width = (totalWidth * totalOrdersDone(order)) / totalOrders(order);
+      progressBarRef.current.style.width = `${width}px`;
+    }
+  }, [order]);
```

Comment:
Isn't calculating a percentage with better than this? No need for refs, and we can use tailwind to define the custom percentage width.

Reply:
I tried at runtime but couldn't make it work with tailwind custom properties. Maybe I'm missing something. 

But from a quick search:

> Tailwind doesn’t include any sort of client-side runtime, so class names need to be statically extractable at build-time, and can’t depend on any sort of arbitrary dynamic values that change on the client